File: 1er Trimestre/EjerciciosAlgoritmo2/Ejercicio6.py
"""Algoritmo que nos permita introducir
por teclado una serie de números enteros que
finalizara con el número 0, el programa deberá mostrar
por pantalla el número mayor de los introducidos"""

#a y b si a es mayor que B reemplaza B sino no lo reemplaza

numero = float(input("Inserta un número: "))
numeroAlto = numero
while numero != 0:
    if numero <= numeroAlto:
        numero = float(input("Inserta un número: "))
    else:
        numeroAlto = numero
        numero = float(input("Inserta un número: "))
print(numeroAlto)

File: ProgramasVarios/src/ejercicios002/Ejercicio6.java
package ejercicios002;
import javax.swing.*;
public class Ejercicio6 {
 /*Algoritmo que nos permita introducir
  * por teclado una serie de números enteros
  * que finalizara con el número 0, el programa
  * deberá mostrar por pantalla el número mayor 
  *de los introducidos. */
	public static void main(String[] args) {
		int entero, mayor;
		entero = Integer.parseInt(JOptionPane.showInputDialog(null, "Inserta un número entero"));
		mayor = entero;
		do {
			entero = Integer.parseInt(JOptionPane.showInputDialog(null, "Inserta otro número, puedes terminar la secuencia con un 0"));
			if (entero > mayor) {
				mayor = entero;
			}
		}while (entero != 0);
		JOptionPane.showMessageDialog(null, "Tu número mayor es: " + mayor);
		}
}
